docs/example-rover-config.rst

```diff
@@ -0,0 +1,16 @@
+websocket protocol
+====================
+
+Here is an example of a Rover config that the service might expect to
+get from the server.
+
+.. code-block:: json
+
+    {
+       "left-motor-port": "A",
+       "right-motor-port": "B",
+	   "left-ultrasonic-port": 1
```

Comment:
Seems like a mixing of tabs and spaces here. Also, missing commas after each line